src/OpenTelemetry.Api/Trace/TracerNew.cs

```diff
@@ -52,6 +52,7 @@ public TelemetrySpanNew CurrentSpan
         /// <param name="kind">Kind.</param>
         /// <returns>Span instance.</returns>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0027:Public API with optional parameter(s) should have the most parameters amongst its public overloads.", Justification = "Temp justification")]
```

Comment:
@cijothomas , i had to include this suppression

Reply:
```suggestion
        [System.Diagnostics.CodeAnalysis.SuppressMessage("ApiDesign", "RS0027:Public API with optional parameter(s) should have the most parameters amongst it's public overloads.", Justification = "This was agreed on to be the friendliest API surface")]
```